### PR TITLE
Network: add highest LC to diagnostics

### PR DESCRIPTION
**Pull request by reinkrul**
Useful for `data-viewer` to it can skip to the last page/transaction.

### CODE REVIEW

File: network/dag/state_test.go [hunk 1]
```diff
@@ -296,24 +296,44 @@ func TestState_Add(t *testing.T) {
...
+		sort.Strings(lines)
+		actual := strings.Join(lines, "\n")
 
-	assert.Contains(t, actual, fmt.Sprintf("dag_xor: %s", doc1.Ref()))
-	assert.Contains(t, actual, "transaction_count: 1")
-	assert.Contains(t, actual, "failed_events: 0")
+		assert.Contains(t, actual, fmt.Sprintf("dag_xor: %s", doc1.Ref()))
+		assert.Contains(t, actual, "transaction_count: 1")
+		assert.Contains(t, actual, "failed_events: 0")
+		assert.Contains(t, actual, "dag_lc_high: 0")
```
Comment (gerardsn) [inline network/dag/state_test.go:318]: does not really tests if `dag_lc_high` works

**Review by gerardsn — APPROVED** [review]

File: network/dag/state_test.go (hunk 1 above)
Comment (woutslakhorst) [reply network/dag/state_test.go:318]: then don't approve ;)

**Review by gerardsn — CHANGES_REQUESTED** [review]
see other comment

File: network/dag/state_test.go (hunk 1 above)
Comment (gerardsn) [reply network/dag/state_test.go:318]: changes requested

**Review by gerardsn — APPROVED** [review]

**Pull request merged**